### ISSUE TITLE
Cut a release of latest loader

### ISSUE THREAD

**Issue by KevinMartin** [AssemblyScript/loader#3]
Please.

**Comment by dcodeIO** [comment]
The currently released loader should be in sync with the currently released main package - or isn't it? Note that npm has 0.3.0 (that's the latest release of the [prototype](https://github.com/AssemblyScript/prototype)).

**Comment by KevinMartin** [comment]
v0.3.0 is from Aug 28th

**Comment by dcodeIO** [comment]
Yeah, as mentioned, npm only has the prototype as of version 0.3.0. Neither 0.4.0, that is the latest state of the [prototype](https://github.com/AssemblyScript/prototype) before it was discontinued, nor 0.5.0, that is the current state of [NEXT](https://github.com/AssemblyScript/assemblyscript) (now in the main repo) has been released on npm yet. Master of loader and prototype *should* be in sync, though, if you'd like to try it on your own.

**Comment by KevinMartin** [comment]
Would you then be able to release 0.4.0 of both loader and prototype? It fixes a bug I'm having in 0.3.0 of loader.
